core/src/test/scala/org/apache/gearpump/cluster/MasterHarness.scala

```diff
@@ -130,13 +138,24 @@ trait MasterHarness {
     mainObj.getClass.getName.dropRight(1)
   }
 
+  import Constants._
+
   def getMasterListOption(): Array[String] = {
-    Array(s"-D${Constants.GEARPUMP_CLUSTER_MASTERS}.0=$getHost:$getPort",
-    s"-D${Constants.GEARPUMP_HOSTNAME}=$getHost")
+    masterProperties.asScala.toList.map { kv =>
```

Comment:
@kkasravi 
Check above code change at line 65
```
  masterProperties.put(s"${GEARPUMP_CLUSTER_MASTERS}.0", s"$getHost:$getPort")
+    masterProperties.put(s"${GEARPUMP_HOSTNAME}", s"$getHost")
```

We already put ".0" in the property list.